### DISCUSSION TITLE
CompreFace or CodeProject.AI Server?

### DISCUSSION THREAD

**Discussion by felipecrs** [skrashevich/double-take#109] [Q&A] [answered]
I'm really new into this stuff. Do you have any recommendations? My computer can run both.

**Comment by skrashevich** [comment, marked as answer]
if you need only face recognition and you have NVIDIA GPU -- CompreFace will be the best choice (I recommend a single-container version for small installations)
if you want to have object detection functionality too, or want to run on CPU (without GPU) -- I think, ai.server will works better
